lib/phoenix/live_dashboard/live/processes_live.ex

```diff
@@ -35,6 +35,16 @@ defmodule Phoenix.LiveDashboard.ProcessesLive do
     <div class="processes-page">
       <h5 class="card-title">Processes</h5>
 
+      <div class="processes-search">
+        <form phx-change="search" phx-submit="search" class="form-inline">
```

Comment:
good call on change+debounce, lmk what you think about the given debounce value.

I left phx-submit so that when muscle memory kicks in and we press "enter" we avoid a page reload. Maybe there's a cleaner way to achieve this? maybe `phx-submit=""` should discard the event?

Reply:
`phx-submit="search"` as is is actually a good solution since it will instantly submit the query, cancel any denounced changes, and provide user the quick feedback as expected? By the way, this all looks great! 😍😍😍

Reply:
300-500 for debounce feels reasonable?